### PR TITLE
Speedseq SV remove sort and symlink VCF.

### PR DESCRIPTION
**Pull request by jasonwalker80**
1. Skip the sort since the output is VCF.
2. Generate a symlink for the VCF (and index) so the DV2 Dispatcher will find the output file.
3. Increase RAM to 16GB

### CODE REVIEW

File: lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm [hunk 1]
```diff
@@ -67,7 +67,9 @@ sub _detect_variants {
     # Make a symlink to the actual VCF as the DV2 Dispatcher expects an output of svs.hq rather than BED or VCF output
     my ($vcf_file) = glob($self->_sv_staging_output .'*.vcf.gz');
     if ($vcf_file) {
-        symlink($vcf_file,$self->_sv_staging_output);
+        my $rel_path = File::Spec->abs2rel( $vcf_file, $self->_temp_staging_directory ) ;
+        Genome::Sys->create_symlink($rel_path,$self->_sv_staging_output .'.vcf.gz');
+        Genome::Sys->create_symlink($rel_path,$self->_sv_staging_output .'.vcf.gz.tbi');
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm]: Should this be adding a `.tbi` to the `$rel_path` as well?

Comment (jasonwalker80) [inline lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm]: good catch,  thanks tom!

**Comment by tmooney** [comment]
+1 🔋

It might be nice to add a test to make sure the standard files are around after it's done running.  Something like:
```perl
ok(-e $command->sv_output, 'output file created as expected');
```
(I think the test I wrote there may fail.  But that's the idea.)

**Comment by jasonwalker80** [comment]
Thanks for the suggestion @tmooney.  Tests added.

**Comment by susannasiebert** [comment]
+1

**Pull request merged**